site/tsconfig.json

```diff
@@ -7,14 +7,14 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "module": "commonjs",
+    "module": "esnext",
     "moduleResolution": "node",
     "noEmit": true,
     "noUncheckedIndexedAccess": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "es5"
+    "target": "es2017"
```

Comment:
Needed for top-level await. Does not change the ouput of Next.js - [proof](https://hashintel.slack.com/archives/C022217GAHF/p1645189648756359?thread_ts=1645186873.448139&cid=C022217GAHF) (internal)